airflow-core/src/airflow/decorators/__init__.py

```diff
@@ -32,6 +30,7 @@
         "DecoratedMappedOperator": "airflow.sdk.bases.decorator.DecoratedMappedOperator",
         "DecoratedOperator": "airflow.sdk.bases.decorator.DecoratedOperator",
         "TaskDecorator": "airflow.sdk.bases.decorator.TaskDecorator",
+        "TaskDecoratorCollection": "airflow.sdk.bases.decorator.TaskDecoratorCollection",
```

Comment:
That is wrong import!

```suggestion
        "TaskDecorator": "airflow.sdk.bases.decorator.TaskDecorator",
        "TaskDecoratorCollection": "airflow.sdk.definitions.decorators.TaskDecoratorCollection",
```

Reply:
Oh boy. Good catch